examples/flask-web-app/DEMO.md

```diff
@@ -3,12 +3,26 @@
 # Preparation
 * Before the demo:
     * clean out all Docker containers and images, except Phusion
-    * Rebuild: `flyingcloud sysbase; flyingcloud pybase; flyingcloud opencv; flyingcloud app`
-
+    * Rebuild:
+        ```bash
+  flyingcloud --no-push --no-pull sysbase
+  flyingcloud --no-push --no-pull pybase
+  flyingcloud --no-push --no-pull opencv
+  flyingcloud --no-push --no-pull opencv
```

Comment:
Two copies of `opencv`; spurious backquote after `app`.

Maybe we should change the push/pull values in `flyingcloud.yaml` instead.